static/src/stylesheets/module/content/_gallery.scss

```diff
@@ -245,3 +245,322 @@
         }
     }
 }
+
+// New gallery stuff
+
+// The main image needs to take up 100vh and the slim header needs to sit just on top of it.
+.l-header--new-gallery {
+    position: absolute;
+    z-index: 1;
+    right: 0;
+    left: 0;
+}
+
+.content__labels--gallery {
+    border: 0;
+    margin-bottom: 0;
+    width: auto;
+    float: none;
+}
+
+// TODO: When the time comes, remove `.new-gallery` class completely and remove old gallery styling
+.new-gallery {
+    .ad-slot--dark {
+        background-color: $multimedia-support-5;
+    }
+
+    .content__main-column {
+        max-width: none;
+
+        @include mq(tablet) {
+            margin-left: gs-span(3);
+        }
+    }
+
+    @include mq(tablet) {
+        .content__dateline,
+        .content__labels--gallery,
+        .byline,
+        .content__meta-container,
+        .meta__break,
+        .meta__save-for-later,
+        .content__headline,
+        .content__standfirst {
+            margin-left: gs-span(3);
+            margin-right: gs-span(1);
+            max-width: gs-span(8);
+        }
+
+        .byline, .content__dateline {
+            border: 0;
+        }
+
+        .content__standfirst {
+            padding-bottom: 0;
+            padding-top: 0;
+
+            &:before {
+                content: none;
+            }
+        }
+    }
+
+    // overriding defaults
+    .meta__save-for-later {
+        position: relative;
+        left: 0;
+    }
+
+    // TODO: Replace all media tones instead of overwriting the defaults
+    &.tonal--tone-media {
+
+        // Overwriting media tone header so that you can see the image instead
+        .tonal__header {
+            background-color: rgb(0, 0, 0);
```

Comment:
we don't have any other `rgb` colours in the codebase. for the sake of consistency, should they be `#000000`? they'll work fine so it's not a blocker.